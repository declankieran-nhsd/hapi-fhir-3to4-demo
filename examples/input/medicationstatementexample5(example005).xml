<?xml version="1.0" encoding="UTF-8"?><MedicationStatement xmlns="http://hl7.org/fhir">
  <text><status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: example005</p><p><b>basedOn</b>: <a>CarePlan/gpvisit</a></p><p><b>context</b>: <a>Encounter/f203</a></p><p><b>status</b>: entered-in-error</p><p><b>medication</b>: Amoxicillin (product) <span>(Details : {SNOMED CT code &#39;27658006&#39; = &#39;p-Hydroxyampicillin&#39;, given as &#39;Amoxicillin (product)&#39;})</span></p><p><b>effective</b>: 23/01/2014</p><p><b>dateAsserted</b>: 22/02/2015</p><p><b>informationSource</b>: <a>Donald Duck</a></p><p><b>subject</b>: <a>Donald Duck</a></p><p><b>taken</b>: unk</p><p><b>note</b>: Patient indicated that they thought it was Amoxicillin they were taking but it was really Erythromycin</p></div></text><basedOn>
    <reference value="CarePlan/gpvisit"/>
  </basedOn>
  <context>
    <reference value="Encounter/f203"/>
  </context>
  <status value="entered-in-error"/>
  <medicationCodeableConcept>
    <coding>
      <system value="http://snomed.info/sct"/>
      <code value="27658006"/>
      <display value="Amoxicillin (product)"/>
    </coding>
  </medicationCodeableConcept>
  <effectiveDateTime value="2014-01-23"/>
  <dateAsserted value="2015-02-22"/>
  <informationSource>
    <reference value="Patient/pat1"/>
    <display value="Donald Duck"/> 
  </informationSource>
  <subject>
    <reference value="Patient/pat1"/>
    <display value="Donald Duck"/> 
  </subject>
  <taken value="unk"/>
  <note>
    <text value="Patient indicated that they thought it was Amoxicillin they were taking but it was really Erythromycin"/>
  </note>
</MedicationStatement>