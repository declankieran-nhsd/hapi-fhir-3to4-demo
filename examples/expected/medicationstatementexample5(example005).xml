<?xml version="1.0" encoding="UTF-8"?>

<MedicationStatement xmlns="http://hl7.org/fhir">
  <text>
    <status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: example005</p><p><b>basedOn</b>: <a>CarePlan/gpvisit</a></p><p><b>context</b>: <a>Encounter/f203</a></p><p><b>status</b>: entered-in-error</p><p><b>medication</b>: Amoxicillin (product) <span>(Details : {SNOMED CT code '27658006' = 'p-Hydroxyampicillin', given as 'Amoxicillin (product)'})</span></p><p><b>effective</b>: 23/01/2014</p><p><b>dateAsserted</b>: 22/02/2015</p><p><b>informationSource</b>: <a>Donald Duck</a></p><p><b>subject</b>: <a>Donald Duck</a></p><p><b>taken</b>: unk</p><p><b>note</b>: Patient indicated that they thought it was Amoxicillin they were taking but it was really Erythromycin</p></div>
  </text>
  <basedOn>
    <reference value="CarePlan/gpvisit"/>
  </basedOn>
  <context>
    <reference value="Encounter/f203"/>
  </context>
  <extension url="http://hl7.org/fhir/3.0/StructureDefinition/extension-MedicationStatement.status">
    <valueCode value="entered-in-error"/>
  </extension>
  <extension url="http://hl7.org/fhir/3.0/StructureDefinition/extension-MedicationStatement.taken">
    <valueCode value="unk"/>
  </extension>
  <status value="unknown"/>
  <subject>
    <reference value="Patient/pat1"/>
    <display value="Donald Duck"/>
  </subject>
  <dateAsserted value="2015-02-22"/>
  <informationSource>
    <reference value="Patient/pat1"/>
    <display value="Donald Duck"/>
  </informationSource>
  <note>
    <text value="Patient indicated that they thought it was Amoxicillin they were taking but it was really Erythromycin"/>
  </note>
  <effectiveDateTime value="2014-01-23"/>
  <medicationCodeableConcept>
    <coding>
      <system value="http://snomed.info/sct"/>
      <code value="27658006"/>
      <display value="Amoxicillin (product)"/>
    </coding>
  </medicationCodeableConcept>
</MedicationStatement>